config/initializers/acts_as_watchable.rb

```diff
@@ -1,8 +1,13 @@
 # Be sure to restart your server when you modify this file.
 
-# In development and non-eager loaded mode, we need to register the acts_as_watchable models manually
+# For development and non-eager load mode, we need to load models using acts_as_watchable manually
 # as no eager loading takes place
-Rails.application.config.after_initialize do
-  OpenProject::Acts::Watchable::Registry
-    .add(WorkPackage, Message, Forum, News, Meeting, Wiki, WikiPage)
+Rails.application.config.to_prepare do
+  Forum
+  Meeting
```

Comment:
This model is coming from meeting module, so it seems more appropriate to move this line to `modules/meeting/lib/open_project/meeting/engine.rb`, but all modules are loaded through `Gemfile.modules`, so not sure what is better - keeping it here or moving it there.